14/umbraco-commerce/key-concepts/events/list-of-validation-events.md

```diff
@@ -0,0 +1,469 @@
+| ValidateDiscountCreate | Triggered to validate the creation of a new discount. Developers can use this event to enforce rules or validations related to the creation process, ensuring data integrity and adherence to business logic. |
+| ValidateDiscountDateRangeChange | Triggered to validate changes to the date range of a discount. Developers can use this event to enforce rules or validations related to date range changes for discounts, ensuring proper configuration and management. |
+| ValidateDiscountDelete | Triggered to validate the deletion of a discount. Developers can use this event to enforce rules or validations related to the deletion process, ensuring it meets specified criteria or conditions. |
+| ValidateDiscountNameChange | Triggered to validate changes to the name of a discount. Developers can use this event to enforce rules or validations related to the modification of discount names, ensuring clarity and consistency in identification. |
+| ValidateDiscountRewardsChange | Triggered to validate changes to the rewards associated with a discount. Developers can use this event to enforce rules or validations related to reward changes for discounts, ensuring adherence to specified rules or conditions. |
+| ValidateDiscountRulesChange | Triggered to validate changes to the rules associated with a discount. Developers can use this event to enforce rules or validations related to rule changes for discounts, ensuring adherence to specified rules or conditions. |
+| ValidateDiscountSave | Triggered to validate the saving of changes to a discount. Developers can use this event to enforce rules or validations related to the save process, ensuring data integrity and adherence to business logic. |
+| ValidateDiscountTypeChange | Triggered to validate changes to the type of a discount. Developers can use this event to enforce rules or validations related to discount type changes, ensuring consistency and adherence to business rules. |
+| ValidateDiscountUpdate | Triggered to validate updates to a discount. Developers can use this event to enforce rules or validations related to the update process, ensuring data integrity and adherence to business logic. |
+
+### Email Template Payment Events
+
+| **Event** | **Description** |
+|---|---|
+| ValidateEmailTemplateAliasChange | Triggered to validate changes to the alias of an email template. Developers can use this event to enforce rules or validations related to the modification of email template aliases, ensuring clarity and consistency in identification. |
+| ValidateEmailTemplateBccAddressChange | Triggered to validate changes to the Blind Carbon Copy (BCC)  addresses of an email template. Developers can use this event to enforce rules or validations related to BCC address changes for email templates, ensuring proper configuration and management. |
```

Comment:
Strange that this definition isn't accepted 🤔 
Could it be because of the extra space after it? 😅 
```suggestion
| ValidateEmailTemplateBccAddressChange | Triggered to validate changes to the Blind Carbon Copy (BCC) addresses of an email template. Developers can use this event to enforce rules or validations related to BCC address changes for email templates, ensuring proper configuration and management. |
```